python/lsst/ts/m2/csc.py

```diff
@@ -330,19 +332,43 @@ async def _telemetry_loop(self):
 
         self.log.debug("Starting telemetry loop from component.")
 
+        messages_consumed = 0
+        messages_consumed_log_timer = asyncio.create_task(
```

Comment:
Please consider to rename the variable to have the `task_` to make the developers notice this is a `asyncio` task. BTW, please consider to add the related `cancel()` function to be able to release the system resource actively.